webrtc.html

```diff
@@ -3295,6 +3295,10 @@
         <var>connection</var>, run the following steps:</p>
         <ol>
           <li>
+            <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+            <code>true</code>, abort these steps.</p>
+          </li>
+          <li>
```

Comment:
I'm reading this as "whenever we do "queue a task to...", we need to say "queue a task that .... checks whether [[isClosed]] is true and then..."
Relegating fixing that to another PR.